hibernate-osgi/src/test/resources/org/hibernate/osgi/test/testing-bundles.xml

```diff
@@ -7,6 +7,6 @@
 <features name="hibernate-osgi-testing-repository" xmlns='http://karaf.apache.org/xmlns/features/v1.2.0'>
     <feature name='hibernate-osgi-testing' version='5.0.0-SNAPSHOT'>
         <bundle>mvn:com.h2database/h2/1.3.170</bundle>
-        <bundle>mvn:org.ops4j.pax.url/pax-url-aether/2.3.0</bundle>
+        <bundle>mvn:com.sun.activation/javax.activation/1.2.0</bundle>
```

Comment:
2 questions here:
- do we really need that? What's its purpose? My main concern is that if we need to add some necessary dependency for testing, maybe our features are incorrect and are missing a jar? Or is it necessary for the H2 driver?
- if we need that, wondering if `javax.activation:javax.activation-api` wouldn't be enough. Not very fond of having `com.sun` artifacts.

Reply:
You're definitely right. I'm going to move activation bundle to the hibernate-orm feature. I tried to change the dependency from `com.sun.activation:javax.activation:1.2.0` to `javax.activation:javax.activation-api`, but it did not work.